src/templates/layouts/vetCenter/index.tsx

```diff
@@ -0,0 +1,344 @@
+  const PrepareForVisitComponent = ({ visitItems }) => {
+    if (visitItems.length === 0) return null
+    return (
+      <>
+        <h2
+          id="prepare-for-your-visit"
+          className="vads-u-margin-top--0 vads-u-font-size--lg small-screen:vads-u-font-size--xl vads-u-margin-bottom--2"
+        >
+          Prepare for your visit
+        </h2>
+        <p>Click on a topic for more details.</p>
+        <div className="vads-u-margin-bottom--3">
+          {/* TODO: Look at this again after paragraph/basic_accordion work is complete */}
+          <va-accordion bordered id="prepare-for-your-visit-accordion">
+            {visitItems.map((item, index) => (
+              <va-accordion-item
+                key={`prepare-for-your-visit-accordion-item-${index}`}
+                class="va-accordion-item"
+                id={`prepare-for-your-visit-accordion-item-${index}`}
+                header={item.field_header}
+                level="3"
+              >
+                <WysiwygField html={item.field_rich_wysiwyg.processed} />
+              </va-accordion-item>
+            ))}
+          </va-accordion>
+        </div>
+      </>
+    )
+  }
+
+  return (
+    <div className="usa-grid usa-grid-full">
+      <div className="usa-width-three-fourths">
+        <article className="usa-content va-l-facility-detail vads-u-padding-bottom--0">
+          {title && (
+            <>
+              <h1 aria-describedby="vet-center-title">{title}</h1>
+              {officialName && title !== officialName && (
+                <p id="vet-center-title" className="field-official-name">
+                  Also called the {officialName}
+                </p>
+              )}
+            </>
+          )}
+          {introText && (
+            <div className="va-introtext">
+              <p>{introText}</p>
+            </div>
+          )}
+          <va-on-this-page class="vads-u-margin-left--1 vads-u-margin-bottom--0 vads-u-padding-bottom--0"></va-on-this-page>
+
+          {/* Locations and contact */}
+          <h2 id="locations-and-contact-information">
+            Locations and contact information
+          </h2>
+          <div
+            className="region-list usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column
+          small-screen:vads-u-flex-direction--row facility"
+          >
+            <div className="usa-width-two-thirds vads-u-display--block vads-u-width--full">
+              <div>
+                <div className="vads-c-facility-detail">
+                  <section className="vads-facility-detail">
+                    <h3 className="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">
+                      Main Location
+                    </h3>
+
+                    {/* For the ExpandableOperatingStatus widget in vets-website */}
+
+                    {/* TODO: potential change to the vets-website widget to not use facilityId as a prop on div */}
```

Comment:
can you explain further here? it looks like you have the `facilityId` based on the ImageAndStaticMap included below

```
            <ImageAndStaticMap
              image={image}
              facilityId={fieldFacilityLocatorApiId}
            />
```

Reply:
In that case its being passed as a prop to a react component, but in this case its being used as the expected key on an html element.  my thought was that this should instead be a data attribute, but that would require a vets-website (and possibly content-buiild) change.